src/test/java/com/google/api/codegen/testdata/php_test_library.baseline

```diff
@@ -794,4 +796,73 @@ class LibraryServiceTest extends PHPUnit_Framework_TestCase
         $this->assertTrue($grpcStub->isExhausted());
     }
 
+    /**
+     * @test
+     */
+    public function getBigBookTest()
+    {
+        list($grpcStub, $client) = $this->createStubAndClient([$this, 'createMockLibraryServiceImpl'], 'createLibraryServiceStubFunction');
+
+        $this->assertTrue($grpcStub->isExhausted());
+
+        // Mock response
+        $name2 = "name2-1052831874";
+        $author = "author-1406328437";
+        $title = "title110371416";
+        $read = true;
+        $expectedResponse = new Book();
+        $expectedResponse->setName($name2);
+        $expectedResponse->setAuthor($author);
+        $expectedResponse->setTitle($title);
+        $expectedResponse->setRead($read);
+        $anyResponse = new Any();
+        $anyResponse->setValue($expectedResponse->serialize());
+        $completeOperation = new Operation();
+        $completeOperation->setName('operations/getBigBookTest')->setDone(true)->setResponse($anyResponse);
```

Comment:
Do we want to call `setDone(true)` after the actual API call? It is more similar to the actual sequence.